src/semgrep_agent/templates/.semgrepignore

```diff
@@ -12,11 +12,13 @@ env/
 .env/
 venv/
 .venv/
+.tox/
 *.min.js
 
-# Common test directories
+# Common test paths
 test/
 tests/
+*_test.go
```

Comment:
This seems risky in that we might ignore a file someone wants scanned but happened to name test?

Reply:
my understanding is that the go ecosystem standardized on this naming so much, it's build into the `go` command itself; I imagine `go test` would complain if someone named a regular file ending in `_test.go` but I'm only 80% certain.